clustered/integration-test/src/test/java/org/ehcache/clustered/management/AbstractClusteringManagementTest.java

```diff
@@ -324,4 +302,59 @@ protected static String normalizeForLineEndings(String stringToNormalize) {
     return stringToNormalize.replace("\r\n", "\n").replace("\r", "\n");
   }
 
+  private static void sendManagementCallOnEntityToCollectStats() throws Exception {
+
+    Context context = readTopology().getSingleStripe().getActiveServerEntity(tmsServerEntityIdentifier).get().getContext();
```

Comment:
@anthonydahanne : important change: the `StatisticCollectorCapability ` is not on the TMS Entity manageemnt service, at a central point. There is 1 `StatisticCollectorCapability` per server (per tms entity for exactly), and it has access to all exposed server providers and stats.